_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -100,6 +124,30 @@ public function videopress_block_update_meta( $request ) {
 
 		$response_body = json_decode( wp_remote_retrieve_body( $result ) );
 		if ( is_bool( $response_body ) && $response_body ) {
+
+			if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
```

Comment:
A comment might be nice here to explain why we had to create separate code for Jetpack or WPCOM